apps/condo/domains/user/components/containers/AuthLayout.tsx

```diff
@@ -36,7 +36,7 @@ export interface AuthPage extends React.FC {
 
 const FOOTER_LINK_STYLE = { color: colors.sberPrimary[7] }
 const AUTH_PAGES_STYLE = { width: '100%', maxWidth: '500px', paddingLeft: '20px', paddingRight: '20px' }
-const HEADER_STYLE = { background: colors.white, padding: '20px', margin: '0px', width: '100%' }
+const HEADER_STYLE = { background: 'transparent', padding: '20px', margin: '0px', width: '100%' }
```

Comment:
It seems, like solution of the problem is in a wrong way.
The problem is in overlapping of a content part (what we see as clipped h1) with header.
It is a problem of layout, not background.
When I shrink browser window, header goes out of document scroll area.
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/2064830/125298744-1af37e80-e331-11eb-9bbe-1661860c6e66.png">

Correct solution is to prevent overlapping between header and content area.

Reply:
maybe it will be correct in this case to remove fixed position of AuthLayout header?